### ISSUE TITLE
MFT-R not showing up in career mode (1.6.1)

### ISSUE THREAD

**Issue by taterkerman** [shadowmage45/SSTULabs#769]
Subject says it all. The part had a spot in the VAB, but no image (just the gray button). Mouseover does bring up the part text, however, but no part.

**Comment by taterkerman** [comment]
Not showing in sandbox, either. This is not a clean install with just SSTU, however.

**Comment by shadowmage45** [comment]
Using Restock by any chance?  (they delete stock models, which SSTU references...)

**Comment by taterkerman** [comment]
Yes! I just added that to look at it. Need to whitelist, then?

**Comment by shadowmage45** [comment]
I guess?  IDK, I'm staying away from ReStock.

**Comment by taterkerman** [comment]
I'll pull it and recheck.

**Comment by taterkerman** [comment]
Fixed it.

**Comment by blowfishpro** [comment]
I can help create a whitelist

**Comment by taterkerman** [comment]
I might honestly whitelist a few of the stock engines, the new version of the 909 and spark actually look OK, indeed more useful for me than "boattail" or other options in restock. I literally only use a few of the stock parts any more, and usually for little probes.

**Comment by shadowmage45** [comment]
@blowfishpro  
Thanks for the offer :)   Might be good for the near term to have something available for the few stock models that SSTU uses.  In the long run of course I want to replace those models/parts entirely.  Have enough on my plate right now though that I sadly haven't even looked at the official Restock release yet.

Off the top of my head, the few stock parts that I can think of that are used are:
* Radial Fuel Tanks
* NRV engine
* 1.25m Service Bay for prototype sample return container (can skip this one probably)
* Small ISRU drill for prototype surface sampler (can skip this one probably)

**Comment by taterkerman** [comment]
I went to the Restock page here on git, to find the whitelist information, but could not find it.

**Comment by blowfishpro** [comment]
@shadowmage45 do you know if `SSTU_MODEL` nodes like `TankX200-32` in `ModelData-StockFuelTanks.cfg` are still in use?  I can't find any references to them?

**Comment by shadowmage45** [comment]
@blowfishpro I don't think so.  They were for an old part, and I think the configs got left in place after the part was removed.

I am using a few other placeholder stock parts for the structural bits in the MRCS part, but I'm not too worried about those as they are intended as placeholders anyway.
